posts/2023-06-29-OutreachyInternshipExperience/index.qmd

```diff
@@ -0,0 +1,195 @@
+how to create user-friendly tools that can be used by a broad audience.
+
+I am proud of the work I have accomplished so far and grateful for the opportunity 
+to develop important skills during this internship with the aim of using technology 
+to facilitate biotech data processes and analysis.
+
+The miaverse packages are quite interesting because each performs a given set 
+of functions on microbiome data which could  be transversal;
+
+<a href="https://github.com/microbiome/mia">mia</a> package provides functions 
+and workflows examples for analyses of microbiome data. Here we use the 
+TreeSummarizedExperiment  the main container for working with microbiome data. 
+
+<a href="https://github.com/microbiome/miaTime">miaTime</a> mostly associated
+with mia  can be used to analyse time series data for microbial communities. 
+It is also based on the TreeSummarizedExperiment data container.
+
+<a href="https://github.com/microbiome/miaViz">miaViz</a> is used for plotting 
+and visualization of microbiome data. The main container for interfacing is 
+the TreeSummarizedExperiment class.
+
+<a href="https://github.com/microbiome/miaSim">miaSim</a> is used to simulate 
+(longitudinal) data for the benchmarking and analysis of quantitative models 
+of microbial communities
+
+## Atrayee's Experience: Microbiome Study Curation
+
+Hello! I am Atrayee Samanta, an undergraduate student at IIEST Shibpur, India. I
+am currently pursuing a B. Tech in Computer Science and Technology. I love
+reading books, watercolour painting and music.
+
+## How I came to know about Open Source and the Outreachy Internships
+
+When I was just starting out as a sophomore, while exploring about the many ways
+people can make a difference in computer science, I came across the many ways in
+which beginners could contribute to projects that made a difference, however
+small it was. I was fascinated by the world of open-source software and the way
+it operates. What drew me to this community was its collaborative nature. The
+idea that people from all over the world could come together to contribute their
+expertise and collectively create something truly remarkable was truly
+wonderful!
+
+While surfing the web, I came across what seemed to me a brilliant opportunity
+to learn and contribute: the Outreachy internship. What drew me to it was the
+nature of the internship: we had to contribute to projects of our own choice for
+a month, and that meant that whether or not I got the internship, I would
+definitely get a chance to learn and interact with the open-source community!
+Once my initial application period was over, I took a few days to go over the
+many projects listed on the platform, for which we could choose to contribute.
+
+### The Microbiome Study Curation Project at Bioconductor
+
+A lot of the projects were very appealing but what stood out to me was the
+‘Microbiome Study curation’ project; for me, it was quite different to the
```

Comment:
@atrayees Not all of the Bioconductor community is familiar with Outreachy so the project names may be unfamiliar to a lot of readers, so I suggested to the other interns to use project names that might be familiar within the community. You can say, for example, the `"Microbiome Study Curation" project with <a href="bugsigdb.org">BugSigDb</a>`. Note, we use the project name and link in case anyone is interested in learning more.

Reply:
Thanks a lot, I committed the changes.